client/scripts/eslint-format.cjs

```diff
@@ -35,4 +35,4 @@ async function main() {
   fs.writeFileSync(output, html)
 }
 
-main()
+main().then()
```

Comment:
Why?

Reply:
IJ is angry when you don't consume promises, is all.